packages/trader/src/Modules/Trading/Components/Form/TradeParams/Turbos/__tests__/barriers-list.spec.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import userEvent from '@testing-library/user-event';
+import BarriersList from '../barriers-list';
+
+const turbos_barrier_choices = ['16', '33', '40'];
+const mockClickCallback = jest.fn();
+const mockHoverCallback = jest.fn();
+
```

Comment:
const classname = 'trade-container__barriers-table__item';

Reply:
Thank you for suggestion!